### ISSUE TITLE
Models not loading upon connection "Failed to load entity on builder from saved NBT.  Did a pack change?"

### ISSUE THREAD

**Issue by LavaHoleD** [DonBruce64/MinecraftTransportSimulator#1216]
So, I had this issue where whenever I joined my world, immersive vehicles models would not display.
The fix was by teleporting elsewhere and then teleporting near the place where vehicles are again (not directly, or else it will not work) to get the models to load. However this fix is going to be incredibly frustrating when it's going to turn as a public Server.

So I did some investigation. First I removed *any* mod that had partial incompatibilities to check if the error was still present (turns out yep). Then I noticed in the corner of my eye something in the console, which displayed:

```
 [18:17:55] [Server thread/ERROR] [mts]: MTSERROR: Failed to load entity on builder from saved NBT.  Did a pack change?
 [18:17:55] [Server thread/ERROR] [mts]: MTSERROR: null
```

Fun thing is that, this error does not always show up. Looks like it does (at least on the server console) only with the first victim. But the error itself still happens to everyone.

**Comment by DonBruce64** [comment]
So I tracked this down, and I can say that it's not the cause of disappearing vehicles.  It *is* related to disappearing hand-held guns though.

**Comment by DonBruce64** [comment]
Fixed in V21.3.0.
